src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -288,6 +288,12 @@ public string X5t
                 return GetStandardClaim(JwtHeaderParameterNames.X5t);
             }
         }
+        
+        /// <summary>
+        /// Gets the certificate used to sign the token
+        /// </summary>
+        /// <remarks>If the 'x5c' claim is not found, null is returned.</remarks>   
+        public string X5c => GetStandardClaim(JwtHeaderParameterNames.X5c);
```

Comment:
`x5c` header should be `string[]` (see https://tools.ietf.org/html/rfc7515#section-4.1.6)

Reply:
I'll work on that.